mm2src/mm2_tests.rs

```diff
@@ -1523,6 +1527,267 @@ fn test_withdraw_and_send() {
     block_on(mm_alice.stop()).unwrap();
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_withdraw_and_send_segwit_to_segwit() {
```

Comment:
As discussed in Discord please unite this with `test_withdraw_and_send_segwit_to_legacy` to avoid the same UTXO usage.